v3/src/components/map/utilities/map-utils.ts

```diff
@@ -1,8 +1,98 @@
-import {IMapContentModel} from "../models/map-content-model"
-// import {IDataSet} from "../../../models/data/data-set"
-// import {getSharedDataSets} from "../../../models/shared/shared-data-utils"
+import {latLngBounds} from 'leaflet'
+import {isFiniteNonEmpty} from "../../../utilities/math-utils"
+import {IDataSet} from "../../../models/data/data-set"
+import {kLatNames, kLongNames} from "../map-types"
+import {IDataConfigurationModel} from "../../data-display/models/data-configuration-model"
 
 
-export const syncMapModelWithDatasets = (mapModel: IMapContentModel) => {
-  // const datasets = getSharedDataSets()
+export const datasetHasPointData = (dataset: IDataSet) => {
```

Comment:
```suggestion
export const datasetHasLatLongData = (dataset: IDataSet) => {
```
and also add a comment about what the criteria are.